### DISCUSSION TITLE
optix 7.6 now safe to use? Or should I stick with 7.3?

### DISCUSSION THREAD

**Discussion by mpjohn** [dreamworksanimation/openmoonray#149] [General]
Hi All,

In the latest version of the release notes it states that Optix 7.6 is now supported.
In the build notes it still says 7.3 is the version to use.
Is it safer to stick with 7.3 as per the build notes?

I'm getting errors about illegal instructions ( core dumped ) which may be unrelated.

Thanks
M

**Comment by jmahovsky-dwa** [comment]
Optix 7.6 is the correct version, and only supported version.  It appears that we missed updating the version in a few locations to 7.6.  Attempting to use 7.3 will likely result in build errors or crashes.

**Reply by mpjohn** [reply]
Good to know. Thanks!
